Formula/gnuradio.rb

```diff
@@ -38,7 +38,7 @@ class Gnuradio < Formula
   depends_on "pyqt@5"
   depends_on "python@3.9"
   depends_on "qt@5"
-  depends_on "qwt"
+  depends_on "qwt-qt5"
```

Comment:
Can we change this all to `qt` and `pyqt` instead?

Reply:
qwtpolar is now part of qwt.
see: https://sourceforge.net/p/qwtpolar/news/2021/07/qwtpolar-has-become-part-of-the-qwt-library/

Reply:
After taking a look at https://github.com/gnuradio/gnuradio/blob/master/gr-qtgui/CMakeLists.txt. It is clear that it supports onlt qt5 and pyqt5.